ManageRDSessions/ManageRDSessions.psm1

```diff
@@ -48,6 +48,9 @@ function Get-sbRDSession {
     param(
         [CmdletBinding()]
         [Parameter(Mandatory = $false)]
+        [Alias("RdsServer")]
+        [string]$ConnectionBroker,
+        [Parameter(Mandatory = $false)]
```

Comment:
Make sure you leave a blank line between params; easier to read.

Reply:
Ah yes. Thanks and done!